content/en/workflows/trigger.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Trigger a workflow
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "dashboards"
+  tag: "Documentation"
+  text: "Find out more about setting up a dashboard"
+- link: "security_platform/explorer"
+  tag: "Documentation"
+  text: "Find out more about Security Signals"
+- link: "monitors"
+  tag: "Documentation"
+  text: "Find out more about Monitors"
+---
+
+You can trigger a workflow manually, on a schedule, or using a variety of automated methods such as a Datadog Monitor, or a Security Signal Notification Rule. Before you can trigger a workflow using automation, you must associate the workflow with a unique service account. For more information on setting up a service account for your workflows, see [Set up workflows][1].
+
+## Manually trigger a workflow
+
+To trigger a workflow manually:
+1. From the workflow page, click the **Run** button.
+1. Enter the values for existing trigger variables.
+1. When you're ready to run the workflow, click **Save & Run**.
+
+## Manually trigger a workflow from a Dashboard
+
+To trigger a workflow from a Dashboard, add the Run Workflow widget:
+1. From your Dashboard, click **Add Widget**.
+1. Search for `workflows` and add the **Run Workflow** widget.
+1. Under **Select the workflow**, find your workflow in the dropdown menu.
+1. Enter a title for the widget and click **Save**
+
+{{< img src="workflows/trigger-from-dashboard.png" alt="Click Run Workflow to trigger a workflow from Dashboard widget." >}}
+
+To run the workflow:
+1. Click **Run Workflow** on your dashboard widget.
+1. Under **Execution parameters**, enter the required trigger variables for your workflow.
+1. Click **Run** to run the workflow.
+
+## Trigger a workflow from a Monitor
+
+To trigger a workflow from a Monitor:
+1. On the workflow canvas, click **Add an Automated Trigger** and select **@mention**.
+1. Next to **@workflow-**, enter a mention name for the trigger. Your mention name must be unique.
+1. Save your Workflow.
+1. Find the monitor you'd like to use to trigger the workflow and edit it, or create a new monitor.
```

Comment:
Can you be more clear leaving Workflows and navigating to monitors here? Maybe add a step like "Navigate to the monitors list" or something.